tasks-by-users/Rehan-Khan-Engineer/10-responsive-portfolio-website-rehan/assets/css/styles.css

```diff
@@ -0,0 +1,1500 @@
+}
+
+.contact_information {
+  display: flex;
+  margin-bottom: var(--mb-2);
+}
+
+.contact_icon {
+  font-size: 2rem;
+  color: var(--first-color);
+
+  margin-right: var(--mb-0-75);
+}
+
+.contact_title {
+  font-size: var(--h3-font-size);
+  font-weight: var(--font-medium);
+}
+
+.contact_subtitle {
+  font-size: var(--small-font-size);
+  color: var(--text-color-light);
+}
+
+.contact_content {
+  background-color: var(--input-color);
+
+  border-radius: 0.5rem;
+
+  padding: 0.75rem 1rem 0.25rem;
+}
+
+.contact_lable {
+  font-size: var(--small-font-size);
+  color: var(--title-color);
+}
+
+.contact_input {
+  width: 100%;
+
+  background-color: var(--input-color);
+  color: var(--text-color);
+
+  font-family: var(--body-font);
+  font-size: var(--normal-font-size);
+
+  border: none;
+  /* outline: none; */
+  padding: 0.25rem 0.5rem 0.5rem 0rem;
+}
+
+/*==================== FOOTER ====================*/
+
+.footer {
+  padding-top: 2rem;
+}
+
+.footer_container {
+  row-gap: 3.5rem;
+}
+
+.footer_bg {
+  background-color: var(--first-color-second);
+  padding: 2rem 0 3rem;
+}
+
+.footer_title {
+  font-size: var(--h1-font-size);
+  margin-bottom: var(--mb-0-25);
+}
+
+.footer_subtitle {
+  font-size: var(--small-font-size);
+}
+
+.footer_links {
+  display: flex;
+  flex-direction: column;
+  row-gap: 1.5rem;
+}
+
+.footer_link:hover {
+  color: var(--first-color-lighter);
+}
+
+.footer_socials {
+  font-size: 1.25rem;
+  margin-right: var(--mb-1-5);
+}
+
+.footer-social:hover {
+  color: var(--first-color-lighter);
+}
+
+.footer_copy {
+  font-size: var(--small-font-size);
+  color: var(--text-color-light);
+
+  text-align: center;
+
+  margin-top: var(--mb-3);
+}
+
+.footer_title,
+.footer_subtitle,
+.footer_link,
+.footer-social {
+  color: #fff;
+}
+
+/*========== SCROLL UP ==========*/
+
+.scrollup {
+  position: fixed;
+  right: 1rem;
+  bottom: -20%;
+
+  background-color: var(--first-color);
+
+  opacity: 0.8;
+
+  padding: 0 0.3rem;
+
+  border-radius: 0.4rem;
+
+  z-index: var(--z-tooltip);
+
+  transition: 0.4s;
+}
+
+.scrollup:hover {
+  background-color: var(--first-color-alt);
+}
+
+.scrollup_icon {
+  font-size: 1.5rem;
+  color: #fff;
+}
+
+/* Show scroll */
+
+.show-scroll {
+  bottom: 5rem;
+}
+
+/*========== SCROLL BAR ==========*/
+::-webkit-scrollbar {
+  width: 0.6rem;
+
+  background-color: var(--scroll-bar-color);
+
+  border-radius: 0.5rem;
+}
+
+::-webkit-scrollbar-thumb {
+  background-color: var(--scroll-thumb-color);
+
+  border-radius: 0.5rem;
+}
+
+::-webkit-scrollbar-thumb:hover {
+  background-color: var(--text-color-light);
+}
+
+/*==================== MEDIA QUERIES ====================*/
+
+/* For small devices */
+
+@media screen and (max-width: 350px) {
+  .container {
+    margin-left: var(--mb-1);
+    margin-right: var(--mb-1);
+  }
+
+  .nav_menu {
+    padding: 2rem 0.25rem 4 rem;
+  }
+
+  .nav_list {
+    column-gap: 0;
+  }
+
+  .home_content {
+    grid-template-columns: 0.25fr 3 fr;
```

Comment:
Is it working or not? @Rehan-Khan-Engineer

Reply:
Corrected @Tauqeer-Ahmed-99